### ISSUE TITLE
Mslug6 Flycast Atomiswave Shader Preset Crashes Retroarch

### ISSUE THREAD

**Issue by diskogavatron** [Duimon/Duimon-Bezel-Project#1]
Experiencing a strange issue with a single game Metal Slug 6 when using Atomiswave preset. All other Atomiswave games work fine, but after setting 'Atomiswave-[ADV]-[Guest].slangp' as the shader, the game initially works with the shader applied, but upon reset or reload, Retroarch just crashes out. 

I've tried changing the renderer from vulkan to glcore, etc. but the same result occurs and only just for mslug6. 

Using a general Retroarch shader such as 'crt-geom.slangp' works fine with no crash upon reset. I've attached the log in hope that you may be able to potentially identify the issue. 

[retroarch__2023_10_06__08_10_58.log](https://github.com/Duimon/Duimon-Bezel-Project/files/12835786/retroarch__2023_10_06__08_10_58.log)

**Comment by Duimon** [comment]
These are very simple presets that take advantage of wildcards to auto load a graphic. If it works on every other game I'm not sure there is much we can do on this end to solve the issue.

Which core are you using to run the games? Flycast?

It may have to do with increased internal resolution but that usually prevents the preset from loading at all. It could be that you can load the shader manually but the scaling prevents it from loading automatically.

Do you have the same issue using the Standard or Lite preset? 

BTW. You will get my attn more quickly on my Libretro forum thread and the community can also chime in.

https://forums.libretro.com/t/duimon-hsm-mega-bezel-graphics-and-presets-feedback-and-updates/28146

**Comment by diskogavatron** [comment]
Thanks for responding. I have absolutely no idea what has changed since I posted my issue here, but it works now? 

I figured just to ensure I'm answering you as best as possible, I will try it again. I may've had 'Widescreen Hack' activated perhaps which I don't anymore? I do apologise for what appears to have been wasting your time. I will in future post to the Libretro thread instead. 

> Which core are you using to run the games? Flycast?
Yes using the latest Flycast core. 

> Do you have the same issue using the Standard or Lite preset?
Yes I was having the same problem occure with Standard and Lite presets.
